app/src/main/java/com/google/android/samples/socialite/ui/Main.kt

```diff
@@ -82,11 +83,14 @@ fun MainNavigation(
     NavHost(
         navController = navController,
         startDestination = "home",
-        popEnterTransition = {
-            scaleIn(initialScale = 1.1F) + fadeIn()
-        },
         popExitTransition = {
-            scaleOut(targetScale = 0.9F) + fadeOut()
+            scaleOut(
+                targetScale = 0.9f,
+                transformOrigin = TransformOrigin(pivotFractionX = 0.5f, pivotFractionY = 0.5f),
```

Comment:
How does transformOrigin change the animation? I don't see a difference in the animation if I leave it out.

Reply:
The default value for TransformOrigin is at the center (0.5f,0.5f). In this case, since that's our desired effect, we could leave it out, but I think leaving it in shows that it can changed to a different point.